plugins/inputs/cpufreq/cpufreq.go

```diff
@@ -0,0 +1,120 @@
+package cpufreq
```

Comment:
As this only works on linux, please make sure it only builds on linux.
```suggestion
// +build linux
package cpufreq
```
Furthermore, you need to create a file `cpufreq_nonlinux.go` with the following content
```
// +build !linux

package cpufreq
```